src/main/java/Duke.java

```diff
@@ -1,10 +1,272 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.*;
+
 public class Duke {
+    private static final String FILE_PATH = "src/main/java/tasks.txt";
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>(100);
+        loadTasksFromFile(tasks);
+        String logo = "██╗░░░██╗██████╗░██████╗░░█████╗░██╗\n"
+                + "██║░░░██║██╔══██╗██╔══██╗██╔══██╗██║\n"
+                + "██║░░░██║██████╔╝██████╦╝██║░░██║██║\n"
+                + "██║░░░██║██╔══██╗██╔══██╗██║░░██║██║\n"
+                + "╚██████╔╝██║░░██║██████╦╝╚█████╔╝██║\n"
+                + "░╚═════╝░╚═╝░░╚═╝╚═════╝░░╚════╝░╚═╝\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        while (true) {
+            String command = scanner.nextLine();
+            System.out.println("____________________________________________________________");
+            try{
+                if (command.equalsIgnoreCase("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!");
+                    break;
+                } else if (command.equalsIgnoreCase("list")) {
```

Comment:
Maybe it will be better if this part is separated into different functions like addToDoTask, etc. to improve readability, what do you think?